sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py

```diff
@@ -822,10 +824,10 @@ def exportable(self):
 
     @property
     def key_type(self):
-        # type: () -> KeyType
+        # type: () -> Union[str, KeyType]
```

Comment:
`KeyType` is a string, so this could simply be `List[KeyType]`, returning `list[~azure.keyvault.certificates.KeyType]`. Similarly for the others below--all our enums are strings.

Reply:
sounds good, thanks!